### ISSUE TITLE
 Batchsize and lr of HTC+Swin-L

### ISSUE THREAD

**Issue by BinZhu-ece** [SwinTransformer/Swin-Transformer-Object-Detection#101]
Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?

**Comment by chrisagblake** [comment]
I'm also interested in this. Do you mind sharing your config for HTC +Swin-L?
